PWGHF/D2H/Tasks/taskLbReduced.cxx

```diff
@@ -0,0 +1,825 @@
+DECLARE_SOA_COLUMN(DecayLengthXY, decayLengthXY, float);                             //! Transverse decay length of candidate (cm)
+DECLARE_SOA_COLUMN(DecayLengthNormalised, decayLengthNormalised, float);             //! Normalised decay length of candidate
+DECLARE_SOA_COLUMN(DecayLengthXYNormalised, decayLengthXYNormalised, float);         //! Normalised transverse decay length of candidate
+DECLARE_SOA_COLUMN(DecayLengthLc, decayLengthLc, float);                             //! Decay length of Lc-baryon daughter candidate (cm)
+DECLARE_SOA_COLUMN(DecayLengthXYLc, decayLengthXYLc, float);                         //! Transverse decay length of Lc-baryon daughter candidate (cm)
+DECLARE_SOA_COLUMN(ImpactParameterLc, impactParameterLc, float);                     //! Impact parameter product of Lc-baryon daughter candidate
+DECLARE_SOA_COLUMN(ImpactParameterBach, impactParameterBach, float);                 //! Impact parameter product of bachelor pion
+DECLARE_SOA_COLUMN(ImpactParameterProduct, impactParameterProduct, float);           //! Impact parameter product of daughters
+DECLARE_SOA_COLUMN(Cpa, cpa, float);                                                 //! Cosine pointing angle of candidate
+DECLARE_SOA_COLUMN(CpaXY, cpaXY, float);                                             //! Cosine pointing angle of candidate in transverse plane
+DECLARE_SOA_COLUMN(MaxNormalisedDeltaIP, maxNormalisedDeltaIP, float);               //! Maximum normalized difference between measured and expected impact parameter of candidate prongs
+DECLARE_SOA_COLUMN(MlScoreSig, mlScoreSig, float);                                   //! ML score for signal class
+DECLARE_SOA_COLUMN(FlagWrongCollision, flagWrongCollision, int8_t);                  //! Flag for association with wrong collision
+} // namespace hf_cand_lb_lite
+
+DECLARE_SOA_TABLE(HfRedCandLbLites, "AOD", "HFREDCANDLBLITE", //! Table with some Lb properties
+                                                              // B meson features                  hf_cand_lb_lite::M,
+                  hf_cand_lb_lite::M,
+                  hf_cand_lb_lite::Pt,
+                  hf_cand_lb_lite::Eta,
+                  hf_cand_lb_lite::Phi,
+                  hf_cand_lb_lite::Y,
+                  hf_cand_lb_lite::Cpa,
+                  hf_cand_lb_lite::CpaXY,
+                  hf_cand::Chi2PCA,
+                  hf_cand_lb_lite::DecayLength,
+                  hf_cand_lb_lite::DecayLengthXY,
+                  hf_cand_lb_lite::DecayLengthNormalised,
+                  hf_cand_lb_lite::DecayLengthXYNormalised,
+                  hf_cand_lb_lite::ImpactParameterProduct,
+                  hf_cand_lb_lite::MaxNormalisedDeltaIP,
+                  hf_cand_lb_lite::MlScoreSig,
+                  hf_sel_candidate_lb::IsSelLbToLcPi,
+                  // D meson features
+                  hf_cand_lb_lite::MLc,
```

Comment:
in principle you have 2 mass values for the Lc (`pKpi` and `piKp`), unless here you assume that all the candidates have only one hypothesis (as it is also done for the `Bs` now, even though it should be better to modify it)

Reply:
Fixed fellow with the Bs for the moment